share/ncs-package/cmake/NcsConfig.cmake

```diff
@@ -32,14 +32,23 @@ if(NOT NO_BOILERPLATE)
     find_package(NcsToolchain ${NCS_TOOLCHAIN_MINIMUM_REQUIRED} ${EXACT} QUIET)
     if(${NcsToolchain_FOUND})
       message("-- Using NCS Toolchain ${NcsToolchain_VERSION} for building. (${NcsToolchain_DIR})")
+
+      # Handle older NCS Toolchains which doesn't set NCS_TOOLCHAI_PROTOC. Those all have protoc at fixed location.
```

Comment:
```suggestion
      # Handle older NCS Toolchains which doesn't set NCS_TOOLCHAIN_PROTOC. Those all have protoc at fixed location.
```

Reply:
Is this correct, though? Older toolchains had the issue where nanopb would be placed at different directories depending on the OS. This was `${NCS_TOOLCHAIN_PATH}/nanopb` on Windows, `${NCS_TOOLCHAIN_PATH}/opt/nanopb` on Linux, and `${NCS_TOOLCHAIN_PATH}/opt/nanopb/nanopb-0.4.4-macosx-x86`.

Reply:
we are close to code freeze, so decide to clean the code and prepare for new toolchain when it's ready.